src/lib/holocene/scroll-to-top.svelte

```diff
@@ -9,8 +9,7 @@
   <slot />
   <div class="back-to-top" class:hidden>
     <Button
-      class="!py-1.5 !px-1"
-      icon="arrow-up"
+      leadingIcon="arrow-up"
```

Comment:
I don't think this is showing up as intended anymore unfortunately (see [this preview](https://holocene.preview.thundergun.io/story/src-lib-holocene-scroll-to-top-story-svelte?variantId=src-lib-holocene-scroll-to-top-story-svelte-2)) 😞 Perhaps something with other recent changes?

Also wondering if we want it to be an `IconButton` instead? If not, we might want to remove some padding here

![Screenshot 2023-08-29 at 4 32 01 PM](https://github.com/temporalio/ui/assets/15069288/14682118-2512-45a4-b01a-cdab3cd01ec7)

Reply:
I don't think regression was added in this PR, I'm currently unable to get the scroll-to-top button to show up in cloud. I'll make a new ticket to fix it 👍